lib/serial_terminal.pm

```diff
@@ -59,20 +59,39 @@ sub add_serial_console {
     prepare_serial_console();
 
 Wrapper for add_serial_console.
+
+Configure serial consoles for virtio support (root-virtio-terminal and
+user-virtio-terminal).
+
+NOTE: if test plans to use more consoles via VIRTIO_CONSOLE_NUM, it have to
+call add_serial_console() with proper console name (beware different number
+for ppc64le).
 =cut
 
 sub prepare_serial_console {
-    # Configure serial consoles for virtio support
-    # poo#18860 Enable console on hvc0 on SLES < 12-SP2
-    # poo#44699 Enable console on hvc1 to fix login issues on ppc64le
+    script_run('systemctl | grep serial-getty');
```

Comment:
why you doing this two times ? `script_run` won't fail on non-zero value anyway so or replace it with `assert_script_run` if you want to fail or it will fail in next line anyway

Reply:
I used it also for debugging output. I guess from these 2x `record_info()` and 2x plain `script_run()` only single `record_info()` is enough.

Reply:
I now deleted only `script_run()`. If nobody against, I'd prefer to having 2x `record_info()` to see the changes.